schemas/_common-schema.json

```diff
@@ -73,6 +73,61 @@
         }
       }
     },
+    "usr": {
+      "type": "object",
+      "description": "User properties",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier of the user"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the user"
+        },
+        "email": {
+          "type": "string",
+          "description": "Email of the user"
+        }
+      }
+    },
+    "connectivity": {
+      "type": "object",
+      "description": "Device connectivity properties",
+      "required": [
+        "status",
+        "interfaces"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "Status of the device connectivity",
+          "enum": ["connected", "not_connected", "maybe"]
+        },
+        "interfaces": {
+          "type": "array",
+          "description": "The list of available network interfaces",
+          "items": {
+            "type": "string",
+            "enum": ["wifi", "cellular", "ethernet", "loopback", "other"]
+          }
+        },
+        "cellular": {
+          "type": "object",
+          "description": "Cellular connectivity properties",
+          "properties": {
+            "technology": {
+              "type": "string",
+              "description": "The type of a radio technology used for cellular connection"
+            },
+            "carrier_name": {
+              "type": "string",
+              "description": "The name of the SIM carrier"
+            }
```

Comment:
do we really want to provide this level of detail?

Reply:
From experience, this can prove very usefull. I remember a time where we had a bug on all request only for a specific Mobile Network Operator which modifier the headers of all requests they handled. It took us some time to discriminate users who had this error until we found they were all users using the same carrier.

Reply:
ok 🤷‍♂️